HW__2.postman_collection.json

```diff
@@ -0,0 +1,233 @@
+					"protocol": "https",
+					"host": [
+						"rickandmortyapi",
+						"com"
+					],
+					"path": [
+						"api",
+						"location",
+						"{{newID}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "post HW_2_2",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"const moment = require(\"moment\");\r",
+							"const uid = require(\"uuid\");\r",
+							"pm.environment.set(\"date\", moment().format('MMMM Do YYYY'));\r",
+							"pm.environment.set(\"guid\", uid()); \r",
+							"pm.environment.set(\"randomString\", CryptoJS.SHA256(_.random(1000) + '{{$guid}}').toString()); "
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"const ajv = require('ajv');\r",
+							"const ajvInstance = new ajv({logger: console});\r",
+							"const jsonObj = pm.response.json();\r",
+							"const schema = {\r",
+							"\r",
+							"            \"date\": \"string\",\r",
+							"            \"guid\": \"string\",\r",
+							"            \"randomString\": \"string\",\r",
+							"}\r",
+							"pm.test('valid', () => {\r",
+							"    pm.expect(ajvInstance.validate(schema, jsonObj)).to.be.true;    \r",
+							"});\r",
+							"if (ajvInstance.errors){\r",
+							"        console.log(\"Current schema issues:\");\r",
+							"        console.log(ajvInstance.errors);\r",
+							"        }\r",
+							"\r",
+							"\r",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\r\n    \"date\": {{date}},\r\n    \"guid\": {{guid}},\r\n    \"randomString\": {{randomString}}\r\n}"
+				},
+				"url": {
+					"raw": "https://postman-echo.com/post",
+					"protocol": "https",
+					"host": [
+						"postman-echo",
+						"com"
+					],
+					"path": [
+						"post"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "HW_2.3",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"\r",
+							"const jsonObj = pm.response.json();\r",
+							"pm.test(\"Status code is 200\", function () {\r",
+							"    pm.response.to.have.status(200);\r",
+							"});\r",
+							"pm.test(\"Check the Content-Type\", function () {\r",
+							"    pm.expect(pm.response.headers.get('Content-Type')).to.eql(\"application/json; charset=utf-8\");\r",
+							"});\r",
+							"const idMorty= jsonObj.results.find(result => result.name === \"Morty Smith\");\r",
+							"console.log(\"ID of Morty Smith it is \" + idMorty.id);\r",
+							"\r",
+							"const newID = idMorty.id+1;\r",
+							"const newURL = 'rickandmortyapi.com/api/character/' + newID;\r",
+							"console.log (newURL)\r",
+							"\r",
+							"pm.sendRequest(newURL,function(t){\r",
```

Comment:
That request should be triggered if the 1st step query has 200 status code.

For example:
```
if (pm.response.code == 200) {
    pm.sendRequest();
}
```